src/helpers/prettify.ts

```diff
@@ -0,0 +1,14 @@
+export const prettify = (message: unknown) => {
+    let prettified = message
+    try {
+        if (typeof prettified === 'string') {
+            prettified = JSON.parse(prettified)
+        }
+        prettified = JSON.stringify(prettified, null, 2)
```

Comment:
JSON.stringify will not work on error objects. It will return `{}` without any usefull info.